File: euclids_algo.py
def euclids_algo(M,N):
    x = M
    y = N
    max_iter = 100
    GCD = 1

    for i in range(max_iter):
        if x > y:
            x = x - y
        if x < y:
            y = y - x
        if x == y:
            GCD = x
            print(f'Greatest Common Diviser {x}')
            break         
    return GCD
    
euclids_algo(50,15)

File: euclids_algo.js
var euclids_algorithm = function (M, N) {
    var x = M;
    var y = N;
    var max_iter = 100;
    var GCD = 1;
    for (var i = 0; i < max_iter; i++) {
        if (x > y) {
            x = x - y;
        }
        if (x < y) {
            y = y - x;
        }
        if (x == y) {
            GCD = x;
            console.log("Greatest Common Diviser = ".concat(x));
            break;
        }
    }
    return GCD;
};
euclids_algorithm(1500, 165);
